### PR TITLE
Readability on large screens (4k and ultrawide)

### PR DESCRIPTION
**Pull request by BrendanMyers88**
Resolves Issue #98 

- Made competition tile text larger for screens above 1920px. 
- Restricted max-width to 1250px for both competition-tiles and searchbar/advanced search filters. 
- Fixed mobile bug which caused advanced search features dropdown not to work on mobile. 
- Corrected visual bug where arrow did not flip on advanced search being clicked.

[Before Image](https://imgur.com/YSJJKrB.jpg)

[After Image](https://imgur.com/z0KOqhz.jpg)

### CODE REVIEW

File: src/static/riot/search.tag [hunk 1]
```diff
@@ -656,7 +659,12 @@
             padding-bottom 0 !important
 
         .content-desktop
-            margin-top 0
+            margin 10px auto !important
+            max-width 1750px
+
+        .ultrawide
```
Comment (ckcollab) [inline src/static/riot/search.tag]: Should this be a more global class? Maybe on some other pages we'll want this as well, not just search?

File: src/static/riot/search.tag [hunk 2]
```diff
@@ -765,10 +773,13 @@
                 overflow hidden
                 text-overflow ellipsis
                 max-width 90vw
+                display block !important
+                @media screen and (min-width 1921px)
```
Comment (ckcollab) [inline src/static/riot/search.tag]: Where does 1921 come from, is this a common #?

File: src/static/riot/search.tag [hunk 3]
```diff
@@ -813,6 +834,18 @@
             display block
             @media screen and (min-width 500px)
                 display inline-block
+
+        @media screen and (min-width 1921px)
+            *
+                font-size 1.15rem !important
```
Comment (ckcollab) [inline src/static/riot/search.tag]: Whoa, does this mess up anything else? Seems like a very broad change, but I suppose this is how you have to do it to make text larger... makes me feel a bit weird but probably fine

Comment (BrendanMyers88) [reply src/static/riot/search.tag]: I believe this is scoped within the competition tile, so it doesn't effect fonts outside of that element. I had to add font-size changes on header elements to increase readability, for example.

File: src/static/riot/search.tag (hunk 2 above)
Comment (BrendanMyers88) [reply src/static/riot/search.tag]: I was using 1921px to represent 'larger than 1920px'. I changed it to 2560px to comply with wider industry standards as shown in this write-up. https://css-tricks.com/optimizing-large-scale-displays/

File: src/static/riot/search.tag (hunk 1 above)
Comment (BrendanMyers88) [reply src/static/riot/search.tag]: Added an ultrawide class to the base styling template.

**Comment by ckcollab** [comment]
NOTE: Blocked on factory generated competitions not working, should be fixed real soon

**Comment by ckcollab** [comment]
Is this ready to merge @BrendanMyers88 ?

**Pull request merged**